lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -127,6 +127,119 @@ def onReplace(self, *args):
+        self.fromPrimLayout, self.inheritedFromPrimField, self.gotoPrimButton = self._createTextField(
+            'from prim', 'kLabelInheritedFromPrim', 'inArrow.png')
+                
+        # Fill the UI.
+        self._refreshUI()
+
+    def _createTextField(self, longName, uiNameRes, image=None):
+        '''
+        Create a disabled text field group and an optional image button with the correct label.
+        '''
+        uiLabel = getMayaUsdLibString(uiNameRes) if self.useNiceName else longName
+        rowLayout = cmds.rowLayout(numberOfColumns=3, adjustableColumn3=2)
+        with LayoutManager(rowLayout):
+            cmds.text(label=uiLabel, annotation=uiLabel)
+            textField = cmds.textField(annotation=uiLabel, editable=False, enableKeyboardFocus=True)
+            if image:
+                button = cmds.symbolButton(enable=False, image=image)
+            else:
+                button = None
+        return (rowLayout, textField, button)
+
+    def onReplace(self, *args):
+        '''
+        Refresh the UI when the time changes.
+        '''
+        # Nothing needed here since USD data is not time varying. Normally this template
+        # is force rebuilt all the time, except in response to time change from Maya. In
+        # that case we don't need to update our controls since none will change.
+        pass
+
+    def _refreshUI(self):
+        '''
+        Refresh the UI when the data is modified.
+        '''
+        matAPI = UsdShade.MaterialBindingAPI(self.prim)
+        mat, matRel = matAPI.ComputeBoundMaterial()
+        if matRel.GetPrim() == self.prim:
+            self._refreshUIForDirect(mat)
+        else:
+            directMatPath = matAPI.GetDirectBinding().GetMaterialPath()
+            self._refreshUIForInherited(mat, matRel, directMatPath)
+
+    def _refreshUIForDirect(self, mat):
+        '''
+        Refresh the UI when the material is directly on the prim.
+        '''
+        # Note: hide UI elements before filling values.
+        cmds.button(self.gotoPrimButton, command='', e=True)
+        cmds.symbolButton(self.gotoPrimButton, edit=True, enable=False, visible=False)
+        cmds.rowLayout(self.inheritedMatLayout, edit=True, visible=False)
+        cmds.rowLayout(self.fromPrimLayout, edit=True, visible=False)
+
+        self._fillUIValues(mat.GetPath().pathString, '', '')
+
+    def _refreshUIForInherited(self, mat, matRel, directMatPath):
+        '''
+        Refresh the UI when the material is inherited from an ancestor prim.
+        '''
+        direct = directMatPath.pathString if directMatPath else ''
+        inherited = mat.GetPath().pathString
+        fromPath = matRel.GetPrim().GetPath().pathString
+
+        ufePath = ufe.Path([
+            self.item.path().segments[0],
+            ufe.PathSegment(fromPath, mayaUsdUfe.getUsdRunTimeId(), '/')])
+        ufePathStr = ufe.PathString.string(ufePath)
+        command = lambda *_: cmds.select(ufePathStr)
```

Comment:
The icon used usually means that the AE will update to the new object without selecting it. This introduces a new effect from an old and familiar icon. Please double check with UX team as selecting the object might require a new or different icon.

Reply:
Ah, I did not know we could display something new in AE without selecting it. I'll look into what command could achieve that.

Reply:
I have another PR based on this, so I'll do the research and change in that one.

Reply:
Seems to be updateAE. I'll make the change in the next PR